.github/ISSUE_TEMPLATE/security_vulnerability.yml

```
@@ -0,0 +1,179 @@
+name: 🔒 Security Vulnerability
+description: Report a security vulnerability (Private)
+title: "[Security]: "
+labels: ["type: security", "priority: critical", "status: needs-review"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ **IMPORTANT**: If this is a critical security vulnerability, please consider reporting it privately first.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)